libraries/chain/include/eosio/chain/kv_chainbase_objects.hpp

```diff
@@ -63,7 +63,7 @@ namespace config {
    template<>
    struct billable_size<kv_object> {
       static constexpr uint64_t overhead = overhead_per_row_per_index_ram_bytes * 2;
-      static constexpr uint64_t serialized_kv_object_size = sizeof(kv_object) - 2*sizeof(shared_blob) + (8 + 4) * 2; ///< 8 for vector data 4 for vector size
+      static constexpr uint64_t serialized_kv_object_size = sizeof(kv_object) - 2*sizeof(shared_blob) + (4 * 2); ///< 8 for vector data 4 for vector size
```

Comment:
This is good find! Do we need to charge the payer field?

Reply:
Good catch, messed up the math here I'll fix that

Reply:
Actually, I don't think this line needs a change at all since the removal of the database field would be handled by the `sizeof(kv_object)`

Reply:
Todd raised the issue to use sizeof might break consensus in the future. Therefore, the develop branch has changed to use magic number instead. It looks like this branch has not been updated with the changes from develop.

Reply:
I'll check the changes on develop, thanks @huangminghuang